### PR TITLE
Finished navbar and carousel

### PR DESCRIPTION
**Pull request by Zicky78**
-Styled navbar

-added carousel

-styled carousel

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,17 +1,145 @@
...
+		/>
+		<title>Restaurante</title>
+		<link rel="stylesheet" href="css/bootstrap.min.css" />
+		<link rel="stylesheet" href="css/all.min.css" />
+		<link rel="stylesheet" href="css/styles.css" />
+	</head>
+	<body>
+		<!-- NAV -->
+
+		<!-- nav.navbar.navbar-default>.container-fluid>(.navbar-header>(button[type="button"].navbar-toggle.collapsed[data-toggle=collapse][data-target="#mainNav"]>(span.icon-bar)*4)+a.navbar-brand>img[src=img/name-01.png])*1 -->
```
Comment (ayojackay) [inline index.html:17]: you left commented out code here. was this intentional?

File: index.html [hunk 2]
```diff
@@ -1,17 +1,145 @@
...
+						<li><a href="#menu">Menu</a></li>
+						<li><a href="#contact">Contact</a></li>
+					</ul>
+				</div>
+			</div>
+		</nav>
+
+		<!-- Carousel Section -->
+
+		<!-- .container-fluid.wrap>#myCarousel.carousel.slide[data-ride="carousel"]>(ol.carousel-indicators>(li[data-target="myCarousel"][data-slide-to="0"])*3)+(.carousel-inner>(.item.main_img$>h1+p+a[role="button"].btn.btn-lg.btn-primary)*3)+(a[href="myCarousel"].left.carousel-control[role="button"][data-slide="prev"]>span.glyphicon.glyphicon-chevron-left[aria-hidden="true"]+span.sr-only{Previous})*2 -->
```
Comment (ayojackay) [inline index.html:60]: same here. was this intentional?

**Review by ayojackay — APPROVED** [review]
This looks awesome. Though, i saw some commented out code, was not sure if it was needed. If so, you're good to merge. If not, please commit and change.

File: index.html (hunk 1 above)
Comment (Zicky78) [reply index.html:17]: This was intentional to review the emmet lines at a later date

**Pull request merged**